routes/skaters.js

```
@@ -0,0 +1,67 @@
+const router = require('express').Router();
+const Skater = require('../model/skater.js');
+// const bodyParser = require('body-parser').json();
+
+var byId = function(req){
+  return {_id: req.params.id};
+};
+
+function parseValidationMessage (err){
+  var errorMessage = 'Error in your JSON input:\n';
+  for (var i=0; i< Object.keys(err.errors).length; i++){
+    errorMessage += err.errors[Object.keys(err.errors)[i]].message + '\n';
+  }
+  return errorMessage;
+}
+
+router
+  .get('/', (req, res) => {
+    const query = req.query.retired ? {retired: req.query.retired} : {};
+    //tried to look into query string for positions (array)
+    Skater.find(query).select('name number')
+      .then((skaters) => {
```

be sure to include catches on your mongoose queries, if there is an error this will fail silently